acceptance/bundle/validate/include_locations/output.dev.json

```diff
@@ -0,0 +1,100 @@
+{
+  "bundle": [
+    {
+      "column": 3,
+      "file": "databricks.yml",
+      "line": 2
+    }
+  ],
+  "bundle.name": [
+    {
+      "column": 9,
+      "file": "databricks.yml",
+      "line": 2
+    }
+  ],
+  "include": [
+    {
+      "column": 3,
+      "file": "databricks.yml",
+      "line": 5
+    }
+  ],
+  "include[0]": [
+    {
+      "column": 5,
+      "file": "databricks.yml",
+      "line": 5
+    }
+  ],
+  "include[1]": [
+    {
+      "column": 5,
+      "file": "databricks.yml",
+      "line": 6
+    }
+  ],
+  "resources": [
+    {
+      "column": 3,
+      "file": "job.yml",
+      "line": 2
+    }
+  ],
+  "resources.jobs": [
+    {
+      "column": 5,
```

Comment:
this is mostly intended to be read by machines so we could consider a more concise format such as 

`"resources.jobs": ["job.yml", 3, 5]`

Reply:
@fjakobs @denik What's the benefit of a more concise format?

Using an array with elements of different types can be ambiguous. If you care about # of characters in the output, we could make the fields "f", "l", and "c", instead of "file", "line", and "column". I would like to keep the entries an object to avoid any ambiguity or clients re-implementing logic.

Reply:
Good point. We don't need that type of volume though. Here we only care about the locations for resources and not what's underneath. To avoid a volume problem we could already limit the output to max 3 levels, such that `resources.jobs.my_job` is included and not anything underneath. WDYT?

Reply:
We might decide to increase max number of levels later, but then we stuck with the format we designed for 3 levels. I don't have visibility how likely is that though.

Reply:
Discussed offline this format, that is simple but quite compact:

files: ["file1.yml", "file2.yml"]

each location is a tuple [index of file, line, column], e.g. [0, 1, 2]

This could also be for v2.